README.md

```diff
@@ -21,8 +21,52 @@ This is a multi module maven project relying on the [parent `pom.xml`](https://g
+* Create your own client certificate (or for demo purpuse use the provided one) and update the further `server.ssl...` properties accordingly. *(**Note** that `server.ssl.key-store-password` and `server.ssl.key-password` must be the same.)*
+* Change the `client_system_name` property to your system name. *(**Note** that it should be in line with your certificate common name e.g.: when your certificate common name is `my_awesome_client.my_cloud.my_company.arrowhed.eu`, then your system name is  `my_awesome_client`)*
+* Adjust the Service Registry Core System location by the `sr_address` and `sr_port` properties.
+* In case of a provider you have to set it's web-server parameters by the `server.address` and `server.port` properties.
+* In case of a consumer decide whether it should act as a web-server or not. If yes, then set the `spring.main.web-application-type` to 'servlet' and set the further server parameters like in the provider case. If not, just left these properties untouched.
+
+##### (2nd) package structure
+All the provided skeleton classes are located in the child packages of `eu.arrowhead` base package.
+* You can create your own classes  under this base package or
+* You can create your own packages like `com.my_company.my_awesome_project` to organize the skeleton and the application code separated. In the latter case if you wish to use spring beans at your custom packages, then you have to let spring framework known about your base package(s). This can be managed by adding the base package name(s) as a string value(s) in the `@ComponentScan` annotaion of the application's `Main.class` *(**Look for the 'TODO' mark** within the main class)*.
+
+##### (3rd) security configuration
+The skeletons provide a built-in arrowhed framework compatible security configuration located in `eu.arrowhead.client.skeleton.consumer|provider.security` package.
+* The `ConsumerSecurityConfig.class` and the `ProviderSecurityConfig.class` extend the `DefaultSecurityConfig.class` which is imported by the client-library dependency and responsible for setting the `server.ssl.enabled` property declared in the `application.properties`. *(**Note** that the `ConsumerSecurityConfig.class` is became effective only when your consumer is a web-server.)*
+* The `ConsumerAccessControlFilter.class` and the `ProviderAccessControlFilter.class` extend the `AccessControlFilter.class` which is imported by the client-library dependency and responsible for setting the security level based on the `application.properties`.
+  -  `ConsumerAccessControlFilter.class` is became effective only when your consumer is a web-server and `server.ssl.enabled` property is true. This filter is responsible for validating whether the received https request is coming from one of the local cloud's client based on its certificate. *(**Look for the 'TODO' mark** within this class if you want to implement additional access rules.)*
```

Comment:
-  `ConsumerAccessControlFilter.class` is ~became~ effective only when your consumer is a web-server and `server.ssl.enabled` property is **set to** true. This filter is responsible for validating whether the received ~https~ **HTTPS** request is coming from one of the local cloud's client based on its certificate. *(**Look for the 'TODO' mark** within this class if you want to implement additional access rules.)*